server/punishments.ts

```diff
@@ -363,6 +369,36 @@ export const Punishments = new class {
 		return FS(SHAREDIPS_FILE).write(buf);
 	}
 
+	/**
+	 * sharedips.tsv is in the format:
+	 * IP, type (in this case always SHARED), note
+	 */
+	async loadSharedIpBlacklist() {
+		const data = await FS(SHAREDIPS_BLACKLIST_FILE).readIfExists();
+		if (!data) return;
+		for (const row of data.split("\n")) {
+			if (!row || row === '\r') continue;
```

Comment:
```suggestion
		for (const row of data.replace('\r', '').split("\n")) {
			if (!row) continue;
```
This way, carriage returns won't get appended to your notes.